terraform/aws/aws.tf

```diff
@@ -290,6 +324,26 @@ resource "aws_security_group" "ui" {
   }
 }
 
+resource "aws_security_group" "edge" {
```

Comment:
whoops! I had made a partial change to make a "common" SG that included 22. I guess that's a bug.